app/build.gradle

```diff
@@ -29,6 +29,12 @@ android {
     }
 }
 
+configurations.all {
+    resolutionStrategy {
+        forcedModules = ['com.android.support:support-annotations:25.1.0']
```

Comment:
can you add `com.android.support:support-annotations:25.1.0` to Dependencies.kt file? Use it as you did with test_rules?

Reply:
Can you just address this in a commit and then I will try to squash the commits myself on the GitHub interface. @mohak1283

Reply:
Okay will do this @isabelcosta :+1: